src/pudl/output/eia.py

```diff
@@ -5,7 +5,6 @@
 from dagster import Field, asset
 
 import pudl
-from pudl.helpers import drop_all_null_records_with_multiindex
```

Comment:
hm are we no longer using this helper function at all? maybe we should delete it altogether.